tools/ci_build/github/azure-pipelines/orttraining-py-packaging-pipeline-torch182-cuda102.yml

```diff
@@ -1,12 +0,0 @@
-trigger: none
-
-stages:
-- template: templates/py-packaging-training-cuda-stage.yml
-  parameters:
-    build_py_parameters: --enable_training --update --build
-    torch_version: '1.8.2'
```

Comment:
Should the old docker files be deleted too?

Reply:
Yes, I believe I've deleted all the old dockerfiles. Let me know if I missed any.